applications/TrilinosApplication/tests/cpp_tests/strategies/builder_and_solvers/test_builder_and_solver.cpp

```diff
@@ -611,10 +611,207 @@ namespace Kratos::Testing
                 auto pnode2 = rModelPart.pGetNode(2);
                 rModelPart.CreateNewMasterSlaveConstraint("LinearMasterSlaveConstraint", 1, *pnode1, DISPLACEMENT_Y, *pnode2, DISPLACEMENT_Y, 1.0, 0.0);
             }
+        }
 
-            // Compute communication plan and fill communicator meshes correctly
-            ParallelFillCommunicator(rModelPart, rDataCommunicator).Execute();
+        // Compute communication plan and fill communicator meshes correctly
+        ParallelFillCommunicator(rModelPart, rDataCommunicator).Execute();
+    }
+
+    /**
+    * @brief It generates a truss structure with an expected solution
+    */
+    static inline bool StructuralTestBuilderAndSolverDisplacement(
```

Comment:
I think it does not need the `inline` qualifier as it is only used in this source file.